src/token.rs

```diff
@@ -0,0 +1,59 @@
+use jsonwebtoken::{decode, encode, Algorithm, DecodingKey, TokenData, EncodingKey, Header, Validation};
```

Comment:
This is coming up as undeclared. I'm guessing you forgot to update the `Cargo.toml` to include `jsonwebtoken`?

Reply:
Sorry, I will update the `cargo.toml` to include `jsonwebtoken`.